### DISCUSSION TITLE
QoL change for the mod select

### DISCUSSION THREAD

**Discussion by Zockerboi** [ppy/osu#26142] [Ideas] [answered]
Hey everyone! I try for quite a while now osu!lazer and i found maybe a "issue" for the mod select. When you search for a mod like for example "Hidden" and then after you selected the mod, the search bar needs to be cleared by hand. I think after the mod was selected, it would be nice that the search bar will be cleared automatically for another mod.

https://github.com/ppy/osu/assets/85823953/afbc4b7f-25ed-4b30-9b2b-dea6081c9372

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/26178
